CONTRIBUTING.md

```diff
@@ -1,9 +1,37 @@
-# How to Contribute
+# Contributing to kpt-functions-catalog
 
-We'd love to accept your patches and contributions to this project. There are
+We'd love to accept your contributions to this project. There are
 just a few small guidelines you need to follow.
 
-## Contributor License Agreement
+## Community Guidelines
+
+This project follows [Google's Open Source Community Guidelines]
+and a [Code of Conduct].
+
+## How to Contribute
+
+This catalog contains configuration functions to fetch, display, customize,
+update, validate, and apply Kubernetes configuration. Contribute functions
+to this catalog by making the following changes:
+
+1. Make the code changes implementing the function and any unit tests. The code
+   belongs under [functions/] in the appropriate directory.
+2. Create an example of how to run your function and put it under [examples/].
```

Comment:
Currently, `examples` dir does not exist.

Reply:
Created an examples directory with helm template example